controllers/internalrequest/adapter.go

```diff
@@ -86,6 +110,26 @@ func (a *Adapter) EnsurePipelineRunIsCreated() (reconciler.OperationResult, erro
 	return reconciler.ContinueProcessing()
 }
 
+// EnsureRequestIsAllowed is an operation that will ensure that the request is coming from a namespace allowed
+// to execute InternalRequests. If the Config spec.allowList is empty, any request will be allowed independently of the
```

Comment:
do you mean regardless instead of independently? I am not sure what the last sentence means when taking independently into account

Reply:
Regardless is better, yeah.